services/users.js

```diff
@@ -498,6 +498,7 @@ module.exports = class UsersService {
         const payload = {
           jti: uuid.v4(),
           email,
+          loc,
```

Comment:
We should do domain validation to ensure this location is whitelisted before we just encode it.